mission/functions/tasks/primary/fn_task_pri_capture.sqf

```diff
@@ -111,8 +111,8 @@ _taskDataStore setVariable ["build_fob", {
 
         private _possibleBases = para_g_bases inAreaArray [
 		getMarkerPos (_taskDataStore getVariable "taskMarker"),
-		selectMax (getMarkerSize (_taskDataStore getVariable "taskMarker") apply {abs _x}),
-		selectMax (getMarkerSize (_taskDataStore getVariable "taskMarker") apply {abs _x}),
+		selectMax ((getMarkerSize "activeZoneCircle") apply {abs _x}),
+		selectMax ((getMarkerSize "activeZoneCircle") apply {abs _x}),
```

Comment:
allows FOBs to be built within the whole AO zone.

Reply:
getting the marker size from `(_taskDataStore getVariable "taskMarker")` was bugging out which is why I've switched it back